packages/front-end/components/Radix/Badge.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Badge as RadixBadge } from "@radix-ui/themes";
+import { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+import { RadixColor } from "@/components/Radix/HelperText";
+
+type Props = {
+  content: string;
+  color?: RadixColor;
+  variant?: "solid" | "soft" | "surface" | "outline";
+  highContrast?: boolean;
+  radius?: "none" | "small" | "medium" | "large" | "full";
```

Comment:
Looking at the Figma file, I think we can restrict these options a bit:
- Only support `solid` and `soft` variants
- Remove `radius` and `highContrast` props entirely

Also, for color, this is the one component where we want to be able to choose any Radix color, not just the limited palette used by the HelperText component.

Reply:
I found a place to replace one and it looks like we're gonna need `radius` if we want to be able to keep a similar look. See screenshot in updated description.

Reply:
We're not trying to keep a similar look to what's in the product today, so I think it's ok to get rid of `radius` and just do what's in the design system Figma